### ISSUE TITLE
Issue with scanning Kokomi

### ISSUE THREAD

**Issue by Blaster20** [Andrewthe13th/Inventory_Kamera#171]
![image](https://user-images.githubusercontent.com/81706579/148776142-95e09631-8b3c-460d-9e07-31c7486e5eb0.png)
The programm stays at the character (Sangonomiya Kokomi) and does nothing, after clicking manuell to the next charater it continues. I think its because of her name display that wierd, like you see in the screenshot

**Comment by Cupcak3** [comment]
Hi there, this issue is hopefully resolved in [the v1.1.1 prerelease](https://github.com/Andrewthe13th/Inventory_Kamera/releases/tag/v1.1.1-prerelease). If it has been, please let us know so we can close this issue.

**Comment by mythera1** [comment]
It might not be the same problem as OP but my Kokomi doesn't get scanned either; she just gets skipped and isn't to be found in the resulting json-file.
This is a screenshot of my Kokomi
![image](https://user-images.githubusercontent.com/36156286/149194537-36480fcb-328f-442e-a98d-7975374ea7ae.png)
![image](https://user-images.githubusercontent.com/36156286/149194643-52f10e8a-cf4a-420b-8401-89e7d84906e1.png)
![image](https://user-images.githubusercontent.com/36156286/149194657-c88ff034-6494-4fe9-9548-9becd193207c.png)

**Comment by Cupcak3** [comment]
@mythera1 Do Kokomi's element and name from the top left appear in the scanner? I don't have her so it's a bit tricky for me to debug in testing so thanks for the screenshots. I take it these were taken in 1600x900?

**Comment by mythera1** [comment]
@Cupcak3 Aye, the display mode is 1600x900.

![image](https://user-images.githubusercontent.com/36156286/149274902-6d1faf05-f9c1-4185-a56b-ec42de5d09a7.png)
It scans it every time. I ran the scanner 5 times over my account and the 5th time it was also in the json file; not sure if this information helps. For all the times it did scan her it did not throw any Exceptions.

It just was weird because my Kaedehara Kazuha always got scanned; his name is split into two lines too.
If you need any other screenshots let me know.

Thanks for your great work btw!

**Comment by Cupcak3** [comment]
I'm a little confused on what you're describing your problem as. Your screenshot of Kamera shows it successfully scanned all the parts of your character. Are you saying you're just not finding her in the resulting exported JSON?

**Comment by mythera1** [comment]
Yes, most of the times I scan she isn't to be found in the resulting exported JSON file. Only after trying some more scans, she appeared there.

**Comment by Cupcak3** [comment]
Interesting. It's possible that the character validation check isn't being passed sometimes and it looks like we are forgetting to output errors detailing that. Thanks for bringing it to my attention. I'll see where we can tighten up code to ensure these checks are passed as well as add more error details. For now I'm going to close this issue as it was originally about the scanner hanging. You may open a new issue with your details if you wish to be notified when we can get it fixed.
